addon/helpers/ember-sparkles/axis.js

```diff
@@ -10,8 +10,13 @@ export function emberSparklesAxis([ scale ], { position, tickFormat, ticks, widt
 
   let result = axisFn().scale(scale);
 
-  if (tickFormat) {
-    result.tickFormat(timeFormat(tickFormat));
+  switch(typeof tickFormat) {
```

Comment:
Just chatted with Zach about this. Thinking around this was to protect against an input that is neither a string nor a function. The suggestion above would allow something to be invoked that was not a function (i.e. an integer) thus throwing an error. In my initial pass here if it was something other than string or function it would keep the default format of the date string - with this suggestion a "tickFormat is not a function" error would be triggered.

@taras thoughts on this?